src/Libraries/Microsoft.Extensions.AI/ChatCompletion/ChatClientStructuredOutputExtensions.cs

```diff
@@ -43,18 +43,18 @@ public static Task<ChatResponse<T>> GetResponseAsync<T>(
         this IChatClient chatClient,
         IEnumerable<ChatMessage> messages,
         ChatOptions? options = null,
-        bool? useNativeJsonSchema = null,
+        bool? useJsonSchema = null,
         CancellationToken cancellationToken = default) =>
-        GetResponseAsync<T>(chatClient, messages, AIJsonUtilities.DefaultOptions, options, useNativeJsonSchema, cancellationToken);
+        GetResponseAsync<T>(chatClient, messages, AIJsonUtilities.DefaultOptions, options, useJsonSchema, cancellationToken);
 
     /// <summary>Sends a user chat text message, requesting a response matching the type <typeparamref name="T"/>.</summary>
     /// <param name="chatClient">The <see cref="IChatClient"/>.</param>
     /// <param name="chatMessage">The text content for the chat message to send.</param>
     /// <param name="options">The chat options to configure the request.</param>
-    /// <param name="useNativeJsonSchema">
+    /// <param name="useJsonSchema">
     /// Optionally specifies whether to set a JSON schema on the <see cref="ChatResponseFormat"/>.
     /// This improves reliability if the underlying model supports native structured output with a schema, but may cause an error if the model does not support it.
-    /// If not specified, the underlying provider's default will be used.
+    /// If not specified, the default value is <see langword="true" />.
```

Comment:
I assume we're keeping it as a `bool?` so that in the future we have wiggle-room around this default, right? e.g. if we add a discovery mechanism in the future. Is there a different wording here would could use to make it clear that it's up to the system but it errs on the side of true?

Reply:
I could change this back to something more like the previous phrasing if that adds legitimacy to updating this later, e.g.

```suggestion
    /// If not specified, the a default value may be determined by the provider or model.
```

Reply:
Maybe just "If not specified, the default value is determined by the implementation; if a specific value must be guaranteed, it should be specified by the caller", or something like that?

I'm probably also overthinking it. Go with whatever you prefer.